packages/OcLaunch/OcLaunch.0.1.3/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+name: "OcLaunch"
```

Comment:
You may want to lowercase the name — most of (if not all) the opam package names are lowercase.